### ISSUE TITLE
`OpenCombine.Publisher` to `Combine.Publisher`

### ISSUE THREAD

**Issue by sbertix** [broadwaylamb/OpenCombine#205]
Hey 👋, 

Is there any way to turn an `OpenCombine.Publisher` into a `Combine.Publisher` "by default"? 

I couldn't find it in the library and, although I realize it would not fit in the whole "1-to-1" porting vision for **OpenCombine**, and a custom implementation is trivial tbh, I feel like it might be a nice addition.

**Comment by MaxDesiatov** [comment]
What's the use case for this? On platforms where Combine is available one would rather use that instead of OpenCombine, and then there's no need to bridge them to each other.

**Comment by sbertix** [comment]
tldr: I feel like it would make the library much more viable when writing libraries/packages, in its own right, not just as a **Combine** "bridge". 

Right now **OpenCombine** makes sense if your project needs to be available on Linux or if you're supporting pre-**Combine** platform versions DIRECTLY on it. 
If you're writing an external library (wrt the end project), it doesn't make things better, as, even with `OpenCombineShim` sharing namespaces and definitions, you're still limited to either "post-Combine" platform versions or Linux. 

Adding an explicit bridge between the two `Publisher`s would allow, for instance, to write a library relying entirely on `OpenCombine`, without having to worry about platform environments and versions, using it like a "regular" standalone reactive framework, and letting the final user decide how to deal with results. 
No code duplication, no `#if canImport`, and even more importantly, no `if #available`, just actual content. 

I know this is far beyond the scopes of this project, but I feel like it would help a lot in making this feel more than just (an amazingly executed) proof of concept.

**Comment by ddddxxx** [comment]
So eventually OpenCombine falling down the rabbit hole...

@sbertix This may help you: https://forums.swift.org/t/combine-backward-deployment-and-linux/37855
and this, if you would like to implement a OpenCombine bridge: https://github.com/cx-org/CXBridge

@MaxDesiatov Do you consider adding `OpenCombineShim`, just like `CXShim` but default to OpenCombine? I personally don't use OpenCombine but I believe it's good for the ecosystem.

**Comment by sbertix** [comment]
> if you would like to implement a OpenCombine bridge: https://github.com/cx-org/CXBridge

I'm not really relying on alternative implementations of **Combine**, I was just tinkering a bit and got "frustrated" with all the missed opportunities. 

> This may help you: https://forums.swift.org/t/combine-backward-deployment-and-linux/37855

I just wish I had found this sooner, as it's exactly what I intended to say, just written properly hahaha 💪

Thanks @ddddxxx
